roles/docker/templates/policy.json.j2

```diff
@@ -0,0 +1,3 @@
+{"name":"policy_1","users":["core", "root"],"actions":[""]}
+{"name":"policy_2","users":["service_account"],"actions":["container_logs","container_top"]}
+{"name":"policy_3","users":["kube"],"actions":["container"], "readonly":true }
```

Comment:
Does this mean if a user is authenticate against kube wont be able to talk to docker? as he wouldn't be in any of these policies?